### PR TITLE
Update configure-siem.md for Datadog

### PR DESCRIPTION
**Pull request by meetrajsinh-crest**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/security/defender-endpoint/configure-siem.md [hunk 1]
```diff
@@ -69,6 +69,15 @@ Using the Microsoft 365 Defender Add-on for Splunk that supports:
 
 For more information on the Microsoft 365 Defender Add-on for Splunk, see [splunkbase](https://splunkbase.splunk.com/app/4959/).
 
+### Datadog
+
+Microsoft 365 Defender for Endpoint Integration with Datadog supports:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-siem.md]:
```suggestion
Microsoft 365 Defender for Endpoint integration with Datadog supports:
```

File: microsoft-365/security/defender-endpoint/configure-siem.md [hunk 2]
```diff
@@ -69,6 +69,15 @@ Using the Microsoft 365 Defender Add-on for Splunk that supports:
...
+
+- Ingesting Microsoft Defender for Endpoint alerts and Incidents
+- Dashboards that enable monitoring metrics across Endpoint, Threats & Vulnerabilities, and Software
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-siem.md]:
```suggestion
- Ingesting Microsoft Defender for Endpoint alerts and incidents
- Dashboards that enable monitoring metrics across endpoints, threats and vulnerabilities, and software
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @meetrajsinh-crest, kindly commit the suggested changes. Many thanks.

File: microsoft-365/security/defender-endpoint/configure-siem.md (hunk 1 above)
Comment (meetrajsinh-crest) [reply microsoft-365/security/defender-endpoint/configure-siem.md]: Updated commit id [71571b9](https://github.com/MicrosoftDocs/microsoft-365-docs/pull/9189/commits/71571b9ef3a1b2ca10a97f6c1482ce89735fa554)

File: microsoft-365/security/defender-endpoint/configure-siem.md (hunk 2 above)
Comment (meetrajsinh-crest) [reply microsoft-365/security/defender-endpoint/configure-siem.md]: Updated in commit id [100e518](https://github.com/MicrosoftDocs/microsoft-365-docs/pull/9189/commits/100e51816e641f96ca1ccf49d756bbd7bae76adb)

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**